SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorPackageAssembly.cpp

```diff
@@ -0,0 +1,242 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialGDKEditorPackageAssembly.h"
+
+#include "Async/Async.h"
+#include "Framework/Notifications/NotificationManager.h"
+#include "Misc/App.h"
+#include "Misc/FileHelper.h"
+#include "UnrealEdMisc.h"
+
+#include "SpatialGDKEditorModule.h"
+#include "SpatialGDKEditorSettings.h"
+#include "SpatialGDKServicesConstants.h"
+#include "SpatialGDKServicesModule.h"
+#include "SpatialGDKSettings.h"
+
+DEFINE_LOG_CATEGORY(LogSpatialGDKEditorPackageAssembly);
+
+#define LOCTEXT_NAMESPACE "SpatialGDKEditorPackageAssembly"
+
+namespace
+{
+	const FString SpatialBuildExe = FSpatialGDKServicesModule::GetSpatialGDKPluginDirectory(TEXT("SpatialGDK/Binaries/ThirdParty/Improbable/Programs/Build.exe"));
+	const FString Linux = TEXT("Linux");
+	const FString Win64 = TEXT("Win64");
+}
```

Comment:
Nit: // anonymous namespace

Reply:
Just to ask the question...
We added the dev_login tag to to spatial constants. I wonder if "Linux" and "Win64" might also be more suited to spatial constants?

Reply:
I don't think Linux and Win64 belong in SpatialConstants because they don't really have anything to do with Spatial.